setup.py

```diff
@@ -94,14 +94,16 @@ def read(*names, **kwargs):
             'deepdiff',
             'pytest',
             'pytest-cov',
+            'tox',
```

Comment:
hmmm... this is going to be a bit odd here, `tox` itself uses this to install its testing requirements. `tox` probably shouldn't be installing `tox` everytime `tox` is called...

Reply:
We can remove. I only considered the manual use case (which I've only ever done once).

Reply:
```suggestion

```